src/table/utils/__tests__/handle-accessiblity.spec.ts

```diff
@@ -417,4 +417,34 @@ describe('handle-accessibility', () => {
       expect(announce).toHaveBeenCalledWith({ keys: ['SNTable.SelectionLabel.NotSelectedValue'] });
     });
   });
+
+  describe('handleKeyToCopyCell: ', () => {
+    let cellValue: Cell;
+    beforeEach(() => {
+      cellValue = {
+        qText: '251',
+      } as Cell;
+    });
+    it('should copying value to clipboard successfully', () => {
+      Object.assign(navigator, {
+        clipboard: {
+          writeText: jest.fn((value) => value),
+        },
+      });
```

Comment:
this is not a proper mock of `writeText`. have you tried using the real function? If that works you can test it by running the method`readText` to see if the text is on the clipboard

Reply:
I couldn't make it work without mocking, properties will be undefined.